src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -941,192 +1024,205 @@ void func_80B161C0(EnTa* this, PlayState* play) {
-            func_80B13AA0(this, func_80B161C0, func_80B16938);
+            EnTa_SetupAction(this, EnTa_WaitForPlayCuccoGameResponse, EnTa_AnimRunToEnd);
         }
     }
 
-    if (this->unk_2E0 & 0x10) {
-        this->unk_2E0 |= 1;
+    if (this->stateFlags & TALON_STATE_FLAG_ANIMATION_FINISHED) {
+        this->stateFlags |= TALON_STATE_FLAG_TRACKING_PLAYER;
     }
 }
 
-void func_80B1642C(EnTa* this, PlayState* play) {
+void EnTa_TalkAfterCuccoGameWon(EnTa* this, PlayState* play) {
     if ((Message_GetState(&play->msgCtx) == TEXT_STATE_EVENT) && Message_ShouldAdvance(play)) {
         if (Inventory_HasEmptyBottle()) {
             Message_CloseTextbox(play);
-            this->unk_2E0 |= 2;
-            func_80B13AA0(this, func_80B15E80, func_80B16938);
+            this->stateFlags |= TALON_STATE_FLAG_GIVING_MILK_REFILL;
+            EnTa_SetupAction(this, EnTa_GiveItemInLonLonHouse, EnTa_AnimRunToEnd);
             func_8002F434(&this->actor, play, GI_MILK, 10000.0f, 50.0f);
         } else {
             Message_ContinueTextbox(play, 0x208A);
-            func_80B13AA0(this, func_80B15E28, func_80B16938);
+            EnTa_SetupAction(this, EnTa_TalkGeneralInLonLonHouse, EnTa_AnimRunToEnd);
         }
     }
 }
 
-void func_80B16504(EnTa* this, PlayState* play) {
-    u16 faceReaction = Text_GetFaceReaction(play, 0x18);
+void EnTa_IdleSittingInLonLonHouse(EnTa* this, PlayState* play) {
+    u16 faceReaction = Text_GetFaceReaction(play, TALON_FACE_REACTION_SET);
 
-    func_80B13AAC(this, play);
+    EnTa_SetTextForTalkInLonLonHouse(this, play);
 
-    if (func_80B142F4(this, play, this->actor.textId)) {
+    if (EnTa_TalkToPlayer(this, play, this->actor.textId)) {
         Audio_PlayActorSfx2(&this->actor, NA_SE_VO_TA_SURPRISE);
 
         if (faceReaction != 0) {
-            func_80B14FAC(this, func_80B15E28);
+            EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_TalkGeneralInLonLonHouse);
         } else {
-            SET_INFTABLE(INFTABLE_7E);
+            SET_INFTABLE(INFTABLE_TALKED_TO_TALON_IN_RANCH_HOUSE);
 
             switch (this->actor.textId) {
-                case 0x207E:
-                case 0x207F:
-                    func_80B14FAC(this, func_80B161C0);
+                case 0x207E: // First time talking in Lon Lon House
+                case 0x207F: // Play cucco game
+                    EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_WaitForPlayCuccoGameResponse);
                     break;
-                case 0x208B:
-                    func_80B14FAC(this, func_80B15FE8);
+                case 0x208B: // Play cucco game or buy milk
+                    EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_WaitBuyMilkOrPlayCuccoGameResponse);
                     break;
                 default:
-                    func_80B14FAC(this, func_80B16364);
+                    EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_ContinueTalkInLonLonHouse);
                     break;
```

Comment:
we tend to add line breaks inbetween cases with breaks, it makes it easier to read. you may choose to do that now, or if not we will probably do a larger cleanup of this in the whole codebase at some point